src/btree/bt_vrfy.c

```diff
@@ -46,6 +46,7 @@ static int __verify_row_leaf_key_order(WT_SESSION_IMPL *, WT_REF *, WT_VSTUFF *)
 static int __verify_row_leaf_page_hs(WT_SESSION_IMPL *, WT_REF *, WT_VSTUFF *);
 static const char *__verify_timestamp_to_pretty_string(wt_timestamp_t);
 static int __verify_tree(WT_SESSION_IMPL *, WT_REF *, WT_CELL_UNPACK *, WT_VSTUFF *);
+static int __verify_btree_id_with_meta(WT_SESSION_IMPL *, uint32_t, const char **);
```

Comment:
This isn't necessary, the function itself appears before any calls to it.
If I'm wrong and it is needed, can these be maintained in sorted order, please?